packages/language-support/src/autocompletion/schemaBasedCompletions.ts

```diff
@@ -52,78 +52,116 @@ export const allReltypeCompletions = (dbSchema: DbSchema) =>
   reltypesToCompletions(dbSchema.relationshipTypes);
 
 function intersectChildren(
-  connectedLabels: Map<string, Set<string>>,
```

Comment:
Do you think we could use just one data structure here? We could invert the ingoing, so all effectively become outgoing